typit.el

```diff
@@ -76,10 +76,10 @@
   '((t (:inherit font-lock-constant-face)))
   "Face used to render statistical values after typing.")
 
-(defcustom typit-dict "english.txt"
```

Comment:
Please don't change the default dictionary, it's for users to set which dictionary they want, so it's better to leave it pointing to English dictionary.